### PR TITLE
add cpm2014 jury members

### PR DESCRIPTION
**Pull request by davojta**
Add first jury member for cpm2014 as example for future work

### CODE REVIEW

**Comment by davojta** [comment]
Dont need to merge to master till finishing work with all jury for 2014 year

**Comment by staaas** [comment]
wouldn't we like to have the same size (or at lease aspect ratio) for the photos?

**Comment by davojta** [comment]
@nott you think we should keep photoes in fixture in common styleguide ? 
I will try )

**Comment by staaas** [comment]
@davojta 

that will def simplify the things later.

**Comment by davojta** [comment]
@nott we have discussed that we didnt need styleguide for images in fixtures right now. We can raise the question later with the team

File: results/migrations/0016_add_results_14_data.py [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+    collection_id = Collection.objects.filter(depth=1)[0]
+    juryindex_page = IndexPage.objects.get(slug='jury')
+
+    for item in get_jury_data():
+        photo = Image(title=item['title'], collection=collection_id)
+
+        photo_file = os.path.join(MIGRATION_DIR, item['photo'])
+        photo.file.save(
+            name=item['title'] + os.extsep + item['photo_ext'],
```
Comment (staaas) [inline results/migrations/0016_add_results_14_data.py:43]: it's better to use `item['photo']` here and get rid of `photo_ext` in `data.json`

File: results/migrations/0016_add_results_14_data.py [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+    JuryMemberPage = apps.get_model("results.JuryMemberPage")
+    Collection = apps.get_model('wagtailcore.Collection')
+
+    jury_member_page_ct = get_content_type(apps, 'results', 'jurymemberpage')
+
+    collection_id = Collection.objects.filter(depth=1)[0]
+    juryindex_page = IndexPage.objects.get(slug='jury')
+
+    for item in get_jury_data():
+        photo = Image(title=item['title'], collection=collection_id)
```
Comment (staaas) [inline results/migrations/0016_add_results_14_data.py:39]: we should probably create separate collections for different kinds of images

**Comment by staaas** [comment]
i'm going to update site structure - that will affect this migration too

so i think it's good to merge this as is

LGTM

**Comment by davojta** [comment]
LGTM

**Pull request merged**